onyo/lib/commands.py

```diff
@@ -917,17 +924,18 @@ def onyo_set(inventory: Inventory,
         If a given path is invalid or changes are made that would result in
         renaming an asset, while `rename` is not true, or if `keys` is empty.
     """
-    paths = paths or []
+    paths = paths or [inventory.root]
     if not keys:
         raise ValueError("At least one key-value pair must be specified.")
-
+    if not all(keys.keys()) or any(key.isspace() for key in keys.keys()):
```

Comment:
How about this? ;-)
```suggestion
    if any(not k.strip() for k in keys.keys()):
```

Reply:
Actually, that fails on `None` for example, so I will do something like this, but have to adjust with a second condition still.

Reply:
True. Thx!